EDA/commit.py

```
@@ -0,0 +1,32 @@
+                    python_files.append(os.path.join(root, file))
+
+    if not python_files:
+        print("No python files found in the directory")
+        return
+
+    for file in python_files:
+        print(f"\nAnalyzing file: {file}")
+        for tool in [("black", "black"), ("pylint", "pylint"), ("flake8", "flake8")]:
+            name, cmd = tool
+            print(f"\nRunning {name} on {file}...\n")
+            subprocess.run(f"{cmd} {file}", shell=True)
```

Ruff (0.8.2)

`subprocess` call with `shell=True` identified, security issue

(S602)